<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogLoadWidget</class>
 <widget class="QWidget" name="LogLoadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>488</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="logInfoLabel">
       <property name="text">
        <string>logInfoLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logModeCombo"/>
     </item>
     <item>
      <widget class="QToolButton" name="loadFileTb">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/open_file.svg</normaloff>:/viewer/open_file.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadTb">
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/sync_black.svg</normaloff>:/viewer/sync_black.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MessageLabel" name="messageLabel" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="cornerHolderW" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="cornerW" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="showLogTb">
           <property name="toolTip">
            <string>Show log viewer</string>
           </property>
           <property name="text">
            <string>Show log</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/show_log.svg</normaloff>:/viewer/show_log.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="resComboLabel">
           <property name="text">
            <string>Plot resolution:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="resCombo"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>3</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showFullTb">
           <property name="toolTip">
            <string>Show full period</string>
           </property>
           <property name="text">
            <string>Full period</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/arrow_all.svg</normaloff>:/viewer/arrow_all.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="startLabel">
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="startTe">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="endLabel">
           <property name="text">
            <string>End:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="endTe">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy -MM-dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="LogView" name="logView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="viewTab">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionFoboff">
   <property name="text">
    <string>&amp;Fob off</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/close.svg</normaloff>:/viewer/close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="text">
    <string>&amp;Kill</string>
   </property>
  </action>
  <action name="actionLookup">
   <property name="text">
    <string>&amp;Look up node in tree</string>
   </property>
   <property name="toolTip">
    <string>Look up node in tree</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogView</class>
   <extends>QTreeView</extends>
   <header>LogView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
